src/main/java/com/synopsys/integration/blackduck/codelocation/signaturescanner/ScanBatch.java

```diff
@@ -54,11 +55,12 @@ public static ScanBatchBuilder newBuilder() {
     private final boolean debug;
     private final boolean verbose;
     private final boolean isRapid;
+    private final ReducedPersistence reducedPersistence;
```

Comment:
It took me a while to come up with a theory to explain why some sig scanner options go into BlackDuckOnlineProperties, and others go here in ScanBatch. I guess ScanBatch is for options that apply in offline mode as well as online (where-as BlackDuckOnlineProperties seems to be for online-only options).

Does reducedPersistence apply when Detect runs offline? If so, these changes seem fine to me.

Reply:
Interesting, thanks for the explanation. I think in this case the flag applies whether detect is run in online or offline mode. I used the --blackduck.offline.mode=true setting and ran the same scans I was doing to test online. I grabbed the BDIO and uploaded it and it seems like BD reacts in the appropriate way when I check the database. Going to leave the code as is based on this.